authnz/src/main/java/io/stargate/auth/Scope.java

```diff
@@ -0,0 +1,12 @@
+package io.stargate.auth;
+
+public enum Scope {
+  READ,
+  MODIFY, // includes INSERT and UPDATE of data
+  DELETE, // encompasses deleting a row, dropping a table, or deleting a keyspace
```

Comment:
Would it be worth splitting "row delete" and "table delete" scopes?

Reply:
Good point. Probably should have `DROP` and `DELETE`. I was thinking you could differentiate between the two when defining resources in IAM but unless you're going row level it won't really make sense.